layered-architectures-module-2/Catalog.Application/Categories/Commands/UpdateCategory/UpdateCategoryCommand.cs

```diff
@@ -0,0 +1,41 @@
+﻿using AutoMapper;
+using Catalog.Application.Common.Interfaces;
+using Catalog.Application.Common.Mappings;
+using Catalog.Domain.Entities;
+using MediatR;
+
+namespace Catalog.Application.Categories.Commands.UpdateCategory;
+
+public class UpdateCategoryCommand : IRequest, IMapFrom<Category>
+{
+    public int Id { get; set; }
```

Comment:
it's better to have a separate DTO model for data. So you could extend data and logic (command might contains configuration data) separately

Reply:
You already have CategoryDto for that